### ISSUE TITLE
Combine / disable notifications

### ISSUE THREAD

**Issue by nfkrocky** [humhub-contrib/gallery#36]
Humhub is creating a new notification and an event in "Last activities" for each uploaded image to the gallery. So if a person will upload 40 images at once, everybody will get 40 notifications about it and all the "latest activity" will basically be wiped out because of 40 new events.

It's very distracting if users are uploading a lot of images — all other notifications like new posts and comments are lost in never ending lists like this:
![imgur-2019_07_04-00:20:08](https://user-images.githubusercontent.com/48351741/60626245-d7d1ad00-9df2-11e9-8d31-fad863428904.png)

It would be much more convenient to have notifications like "User created a new Media test.jpg and 39 more" instead of "User created a new Media test1.jpg", "User created a new Media test2.jpg" and so on if images were uploaded to the same gallery at once.

But for now — is it possible to completely disable gallery upload notifications at all?

**Comment by siwa-tkapplmueller** [comment]
Hi, are there any news on this?

**Comment by yurabakhtin** [comment]
@luke- Do you know how to enable notifications on creating new `File`?
I see only Activities on upload/create new `File` because the model `Media` is extended from core `File`:

![activities](https://user-images.githubusercontent.com/6995524/168325268-d36b2af5-e483-457a-90ec-3fb5fcfa1b8b.png)

![notifications](https://user-images.githubusercontent.com/6995524/168325258-124ff93b-7e45-442d-8886-e87eadb7c26d.png)

As I understand we should disable the Notifications and Activities for each creating/uploading of new `Media` and create some new notification like "User uploaded X new files into gallery "Gallery Name"", right?

**Comment by yurabakhtin** [comment]
@luke- New activity for Gallery has been added in the commit https://github.com/humhub-contrib/gallery/pull/100/commits/401a347e2d6f11d1d5f71a2e0fd9ae2d80dbedac:

![gallery_activity](https://user-images.githubusercontent.com/6995524/169985361-73bff3af-e7c9-458d-b081-7371dc81cf4d.png)

---

email:

![gallery_email](https://user-images.githubusercontent.com/6995524/169985431-75e6e4f6-cb36-48d1-a693-499ac6f1e404.png)

---

web notification:

![gallery_notification](https://user-images.githubusercontent.com/6995524/169985527-0756313a-606e-47a9-8923-971ea03e4929.png)

**Comment by Kikketer** [comment]
Is there anything holding up this PR from merging?  I've currently had to put a gallery into it's own space because of the notification noise it creates.

**Comment by luke-** [comment]
@Kikketer Yes, unfortunately we have to find another solution that is also suitable for other modules such as the File module.

https://github.com/humhub/humhub/issues/5334
